app/javascript/controllers/projects/samples/single_delete_controller.js

```diff
@@ -0,0 +1,24 @@
+import { Controller } from "@hotwired/stimulus";
+
+// removes sample from sessionstorage if it was checked/selected then removed using the remove link
+export default class extends Controller {
+  static values = {
+    storageKey: {
+      type: String,
+      default: location.protocol + "//" + location.host + location.pathname,
+    },
+    deleteId: {
+      type: String
+    }
+  }
+
+  clear() {
+    const storageValues = JSON.parse(
```

Comment:
**nitpick:** Instead of this, use a sessionStorage outlet and call the remove item in there.

Reply:
No longer necessary as we discussed moving logic to the `selection_controller`